packages/wallet-apis/src/experimental/actions/requestQuoteV0.ts

```diff
@@ -0,0 +1,52 @@
+import type { Static } from "@sinclair/typebox";
+import {
+  toHex,
+  type Address,
+  type IsUndefined,
+  type Prettify,
+  type UnionOmit,
+} from "viem";
+import type { BaseWalletClient, OptionalChainId } from "../../types.ts";
+import type { wallet_requestQuote_v0 } from "@alchemy/wallet-api-types/rpc";
+import { AccountNotFoundError } from "@alchemy/common";
+
+export type RequestQuoteV0Params<
+  TAccount extends Address | undefined = Address | undefined,
+> = Prettify<
+  OptionalChainId<
+    UnionOmit<
+      Static<
+        (typeof wallet_requestQuote_v0)["properties"]["Request"]["properties"]["params"]
+      >[0],
+      "from"
+    >
+  > &
+    (IsUndefined<TAccount> extends true ? { from: Address } : { from?: never })
```

Comment:
If I understand correctly, this means that if an account is hoisted, you cannot provide a "from" right? 

I wonder if there's a use case where users might actually want to request quotes for other from accounts using the same client. Probably better not to allow that though. Great as is! Just floating a thought.